### ISSUE TITLE
SCNvimStatusLine fails

### ISSUE THREAD

**Issue by kflak** [davidgranstrom/scnvim#21]
Hi,

Getting this when trying to run the statusline command according to the readme:
```
ERROR: Primitive '_NetAddr_SendRaw' failed.
caught exception 'send_to: Invalid argument' in primitive in method NetAddr:sendRaw
RECEIVER:
Instance of NetAddr {    (0x557643c8cbb8, gc=C8, fmt=00, flg=00, set=02)
  instance variables [4]
    addr : Integer 2130706433
    port : Integer 0
    hostname : "127.0.0.1"
    socket : nil
}
PROTECTED CALL STACK:
	Meta_MethodError:new	0x557641465000
		arg this = PrimitiveFailedError
		arg what = caught exception 'send_to: Invalid argument' in primitive in method NetAddr:sendRaw
		arg receiver = a NetAddr(127.0.0.1, 0)
	Meta_PrimitiveFailedError:new	0x55764146b6c0
		arg this = PrimitiveFailedError
		arg receiver = a NetAddr(127.0.0.1, 0)
	Object:primitiveFailed	0x5576409d5ec0
		arg this = a NetAddr(127.0.0.1, 0)
	Meta_SCNvim:sendJSON	0x5576429eab40
		arg this = SCNvim
		arg object = {"status_line":{"server_status":"2e+01% 2e+01% 1255u 22s","level_meter":"-inf dB"}}
		arg vimPort = 0
	a FunctionDef	0x5576420a7900
		sourceCode = "<an open Function>"
	Routine:prStart	0x557642052800
		arg this = a Routine
		arg inval = 18.041459666
CALL STACK:
	MethodError:reportError
		arg this = <instance of PrimitiveFailedError>
	< closed FunctionDef >
		arg error = <instance of PrimitiveFailedError>
	Integer:forBy
		arg this = 0
		arg endval = 0
		arg stepval = 2
		arg function = <instance of Function>
		var i = 0
		var j = 0
	SequenceableCollection:pairsDo
		arg this = [*2]
		arg function = <instance of Function>
	Scheduler:seconds_
		arg this = <instance of Scheduler>
		arg newSeconds = 19.041857162
	Meta_AppClock:tick
		arg this = <instance of Meta_AppClock>
		var saveClock = <instance of Meta_SystemClock>
	Process:tick
		arg this = <instance of Main>
^^ The preceding error dump is for ERROR: Primitive '_NetAddr_SendRaw' failed.
caught exception 'send_to: Invalid argument' in primitive in method NetAddr:sendRaw
RECEIVER: a NetAddr(127.0.0.1, 0)
```

Relevant sections of my init.vim:
```
let g:lightline = {}
let g:lightline.component_function = {
  \ 'server_status': 'scnvim#statusline#server_status',
  \ }

function! s:set_sclang_lightline_stl()
  let g:lightline.active = {
  \ 'left':  [ [ 'mode', 'paste' ],
  \          [ 'readonly', 'filename', 'modified' ] ],
  \ 'right': [ [ 'lineinfo' ],
  \            [ 'percent' ],
  \            [ 'server_status'] ]
  \ }
endfunction
```
System: archlinux-5.0.4-1-ck

**Comment by davidgranstrom** [comment]
@kflak Thanks for the report! 
I backported a fix from PR #6 which hopefully solves your issue, its merged to master now. Please let me know if your problem persists.

**Comment by kflak** [comment]
That seems to sort out the issue! Thanks for the quick fix :-)jj
